packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -2,8 +2,12 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:meta/meta.dart';
+
 import 'basic_types.dart';
+import 'diagnostics.dart';
 import 'print.dart';
```

Comment:
I think you want a blank line to separate imports and exports.

Reply:
done. Was able to remove the export as it has historic from before the FlutterErrorBuilder.